packages/vue/src/components/organisms/SfList/SfList.stories.js

```diff
@@ -26,7 +26,7 @@ storiesOf("Organisms|List", module).add("Common", () => ({
         :key="item.label" 
         :style="{margin: '10px 0'}"
       >
-        <SfMenuItem :label="item.label" :count="item.count"/>
+        <div>{{item.label}}</div>
```

Comment:
```suggestion
        <SfMenuItem :label="item.label" :count="item.count"/>
```
in example you should present commo user scenerio, when user copy story, then it should have right with SFUI elements

Reply:
ok thanks for explanation.